src/v2/guide/class-and-style.md

```diff
@@ -153,7 +153,7 @@ When `isActive` is truthy, the rendered HTML will be:
 
 ### Object Syntax
 
-The object syntax for `v-bind:style` is pretty straightforward - it looks almost like CSS, except it's a JavaScript object. You can use either camelCase or kebab-case (use quotes with kebab-case) for the CSS property names:
+The object syntax for `v-bind:style` is pretty straightforward—it looks almost like CSS, except it's a JavaScript object. You can use either camelCase or kebab-case (use quotes with kebab-case) for the CSS property names:
```

Comment:
This is the only part of the review I disagree with. It's harder on the eyes to have dashes without spaces like this.